src/GslCore/GslCore.fsproj

```diff
@@ -60,7 +56,13 @@
     <Compile Include="GslcProcess.fs" />
     <Compile Include="SeamlessPlugin.fs" />
     <Compile Include="Gslc.fs" />
-    <Compile Include="AssemblyInfo.fs" />
+    <FsLex Include=".\GslLexer.fsl">
+      <OtherFlags>--unicode -o GslLexer.fs</OtherFlags>
+    </FsLex>
+    <FsYacc Include=".\GslParser.fsy">
+      <OtherFlags>--module GslParser -o GslParser.fs</OtherFlags>
+    </FsYacc>
+    <Compile Include="LibraryPartProvider.fs" />
   </ItemGroup>
```

Comment:
Ah, see them here - I think they probably need to be deleted above then, or else they are there twice?  Sorry if I'm just misreading the diff

Reply:
Dunno what's up with this, not sure why these changed.  I'll revert them next week.